### PR TITLE
Wt 342 forgot password timer

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Introducing the timer for forgot password email
- TODO: Make the function actually do something

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-342 #done

### CODE REVIEW

File: packages/shared/src/components/auth/ForgotPasswordForm.tsx [hunk 1]
```diff
@@ -20,7 +21,22 @@ function ForgotPasswordForm({
   onClose,
 }: ForgotPasswordFormProps): ReactElement {
   const [emailSent, setEmailSent] = useState(false);
-  const onSendEmail = () => setEmailSent(true);
+  const [timer, setTimer] = useState(0);
```
Comment (sshanzel) [inline packages/shared/src/components/auth/ForgotPasswordForm.tsx:24]: I think we can create a custom hook here to abstract these conditions. Something like `useTimer` - I think we can utilize it in some areas in our auth system.

**Review by sshanzel — APPROVED** [review]
Just had one comment for reusability, but things look good to me! 🚢

File: packages/shared/src/components/auth/ForgotPasswordForm.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/auth/ForgotPasswordForm.tsx:24]: Good point, definitely something we can use, i'll refactor

**Comment by rebelchris** [comment]
@sshanzel Migrated to a useTimer function, once we merge this, i'll revamp the other PR with this flow.

**Review by sshanzel — APPROVED** [review]
Love the hook! Can't wait to use it myself! 🚢

**Pull request merged**